source/includes/stackpath/_delivery_rules.md

```diff
@@ -49,6 +49,86 @@ Attributes | &nbsp;
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliveryrules?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+
+> Request body example for creating a delivery rule:
+
+```js
+{
+ "name": "rule",
+  "deliveryRuleConditions": [
+  {
+    "trigger": "HEADER",
+    "operator": "MATCHES",
+    "target": "my-header"
+  }],
+  "deliveryRuleActions": [
+    {
+      "actionType": "REDIRECT",
+      "redirectUrl": "https://my-url.com"
+    }]
+}
+```
+
+> The above commands return a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/deliveryrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Delivery Rules which allow you to modify CDN settings based on a number of elements like URL, HTTP headers, and more.
+
+Query Params | &nbsp;
+---- | -----------
+`siteId`<br/>*UUID* | The ID of the site for which to create the delivery rule. This parameter is required.
+
+Required| &nbsp;
+------------------------| -----------
+`name`<br/>*string* | The name of the delivery rule.
+`deliveryRuleConditions`<br/>*Array[[Condition](#stackpath-condition)]* | At least one condition.
+`deliveryRuleActions`<br/>*Array[[Action](#stackpath-action)]* | At least one action.
+
+#### Condition
+
+Required| &nbsp;
+------------------------| -----------
+`trigger`<br/>*Enum* | Trigger for the condition. Possible values: `COOKIE`, `HEADER`, `HTTP_METHOD`, `STATUS_CODE`, and `URL`.
+`operator`<br/>*Enum* | Operator for the condition. Possible values: `EQUALS`, `MATCHES`, and `MATCHES_REGULAR_EXPRESSION`.
+`target`<br/>*String* | The target of the condition.
+`httpMethods`<br/>*Array[Enum]* | HTTP methods for the condition. Possible values are: `GET`, `POST`, `PUT`, `DELETE`, `HEAD`, `PATCH`, and `OPTIONS`. This fields is only used when `trigger` is set to `HTTP_METHOD`.
+
+#### Action
+
+The action attributes needed to be passed will vary depending on the condition being specified.
+
+Possible Attributes| &nbsp;
+------------------------| -----------
+`actionType`<br/>*Enum* | The type of action. Possible values: `ADD_RESPONSE_HEADER`, `ADD_HEADER_TO_ORIGIN`,`ADD_HEADER_TO_CDN`, `NEVER_EXPIRE`, `DO_NOT_CACHE`, `CACHE`, `REDIRECT`, `HIDE_HEADER`, `MODIFY_HEADER`, `SIGN_URL`, and `BYPASS_CACHE`.
+`responseHeaders`<br/>*Array* | Headers for the actions expressed as a key-value pair.
+`originHeaders`<br/>*Array* | Headers for the actions expressed as a key-value pair.
+`cdnHeaders`<br/>*Array* | Headers for the actions expressed as a key-value pair.
+`cacheTtl`<br/>*Number* | Time to live of the cache.
+`redirectUrl`<br/>*String* | The url to redirect to.
+`headerPattern`<br/>*String* | The header to set to modifier the response.
```

Comment:
`The header to set to modify the response` instead? Not sure about the context but the current sentence sounds a little off

Reply:
Good catch it should read `The header to set the modifier in the response`. Stand by.

Reply:
@aaronalmeida Fixed in b64562050547d87c0cb13bc0bea0a6111a62f09d